README.md

```diff
@@ -556,6 +559,48 @@ module.exports = buildWebpackConfig([
   workerLoader(),
 ])
 ```
+
+```js
+// src/workers/myawesome.worker.ts
+
+import { get } from 'lodash'
+// you can import modules in this worker
+
+const ctx: Worker = self as any
+
+// Listen to message from the parent thread
+ctx.addEventListener('message', event => {
+  console.log(event)
+  // Post data to parent thread
+  ctx.postMessage({ data: 'maronn' })
+})
+```
+
+```js
+// src/typings/custom/index.d.ts
+declare module 'worker-loader!*' {
+  class WebpackWorker extends Worker {
+    constructor()
+  }
+
+  export = WebpackWorker
+}
+```
+
+```js
+// src/index.tsx
+import Worker from 'worker-loader!./workers/myawesome.worker.ts'
```

Comment:
don't use custom imports, rather change the worker-loader config like this `test: /\.worker\.(js|ts)$/,`